tiledb/sm/array_schema/array_schema.cc

```diff
@@ -106,6 +106,11 @@ ArraySchema::ArraySchema(
           memory_tracker_->get_resource(MemoryType::ENUMERATION_PATHS))
     , current_domain_(make_shared<CurrentDomain>(
           memory_tracker, constants::current_domain_version)) {
+  // Set timestamp to the user passed-in range, otherwise set to the current
+  // time
+  uint64_t now = utils::time::timestamp_now_ms();
+  timestamp_range_ = timestamp_range.value_or(std::make_pair(now, now));
```

Comment:
Nit: move to initializer list

Reply:
Is this possible if I'm calling `uint64_t now = utils::time::timestamp_now_ms();`?

Reply:
You'll just have to invoke it twice:
```C++
, timestamp_range_(timestamp_range.value_or(std::make_pair(<ts_now()>, <ts_now()>)))
```
This isn't entirely necessary; if you prefer to keep as-is to avoid the second call, that's fine.

Also you've just taught me that `std::optional::value_or()` exists... (lol)

Reply:
I'd like to make it just a single call so that it doesn't accidentally end up with two different timestamps